lib/ansible/modules/monitoring/standup.py

```diff
@@ -0,0 +1,617 @@
+      checks are specified in YAML format. A check contains a command that can be run on the host being verified, and,
+      directives to evaluate the command output. Optionally, another command can be specified with a check to heal the
+      state that the check verifies and it is run if the check fails.
+version_added: "2.5"
+options:
+    checks_file_path:
+        description:
+            - Path to the YAML file in which the checks are defined. For format of checks YAML file and syntax of check
+              refer U(https://github.com/thomastk/ansible-module-standup/blob/master/README.md)
+        required: true
+    checks_format:
+        description:
+            - The format of checks_file_path.
+        required: false
+        choices: ['yaml']
+        default: yaml
+    roles:
+        description:
+            - A role is a tag that indicates the application role of a host and a check can be tagged using those. If
+              roles are specified, only matching roles from checks_file_path are executed by the module.
+        required: false
+    heal_state:
+        description:
+            - Indicates that heal command, if that is specified, be run if a check fails.
+        required: false
+        choices: [true, false]
+        default: false
+    continue_on_failure:
+        description:
+            - Indicates that the validation process will continue if a check fails. A check status can be ignored at
+              the check level also.
+        required: false
+        choices: [true, false]
+        default: true
+notes:
+    - Tested on Mac, Ubuntu and CentOS.
+requirements: ['yaml']
+'''
+
+EXAMPLES = '''
+# Run all the checks specified in a checks file.
+- name: Run all the checks in verify-app-cluster.yml
+  standup:
+    checks_file_path: verify-app-cluster.yml
+
+# Run only checks that are tagged with roles db and web from verify-app-cluster.yml
+- name: Run all the checks in verify-app-cluster.yml
+  standup:
+    checks_file_path: verify-app-cluster.yml
+    roles: db,web
+
+# Run only checks that are tagged with roles db and web from verify-app-cluster.yml and
+# run heal command, check status fails.
+- name: Run all the checks in verify-app-cluster.yml and run heal command, if needed
+  standup:
+    checks_file_path: verify-app-cluster.yml
+    roles: db,web
+    heal_state: true
+
+# A sample check. Multiple checks can be specified in a checks file.
+# Refer U(https://github.com/thomastk/ansible-module-standup/blob/master/README.md)
+# for complete documentation of checks YAML file syntax.
+- name: "Test db 1"
+  description: Check if mysql service is running
+  roles: db
+  command: ps -ef |grep mariadb|wc -l
+  heal: sudo service mariadb start
+  output_compare:
+     type: number
+     value: 3
+     operator: EQ
+'''
+
+RETURN = '''
+checks:
+    description: The checks with results of running check and heal commands.
+    type: dict
```

Comment:
`type: complex`

Have a look at other modules with the above line for how to format this

Reply:
Checked in the reformatted RETURN block.